src/store/types/index.ts

```diff
@@ -4,12 +4,12 @@ import {
   TRemoveFromArrayAction,
   TUpdateInArrayAction,
   TPushIntoArrayAction,
-  IForm,
```

Comment:
Makes sense!

`IForm` is being used in the selectors and reducers so it should still probably be exported from this file somewhere, but you're right that it shouldn't be with the payload actions. It's not an important change though so feel free to merge!